src/components/new-safe/create/steps/ReviewStep/index.tsx

```diff
@@ -79,7 +79,7 @@ const ReviewStep = ({ data, onSubmit, onBack, setStep }: StepRenderProps<NewSafe
 
     const safeAddress = await computeNewSafeAddress(provider, props)
 
-    setPendingSafe({ ...data, saltNonce, safeAddress })
+    setPendingSafe({ ...data, saltNonce, safeAddress, willRelay })
```

Comment:
Just noticed that this and `onSubmit` are both using the same data structure. Instead of spreading in both, lets extract this to a variable.